my_list.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] According to the project requirements, you should, in MyList, implement a method #each that yields successive members of @list and uses the MyEnumerable module. Kindly work on adding that method so the project works as expected 👍🏽

<img width="1024" alt="Screenshot 2023-11-02 at 8 23 55 PM" src="https://github.com/Megagig/Ruby-Enumerable/assets/50941074/d3f3146f-d282-4907-89c5-41183fffbc7f">

Reply:
This change has not yet been made.